File: Solutions/344. Reverse String/344.java
class Solution {
    public void reverseString(char[] s) {
        int start = 0, end = s.length - 1;
        char temp = 'a';
        while(start < end){
            temp = s[end];
            s[end] = s[start];
            s[start] = temp;
            start++;
            end--;
        }
    }
}

File: Solutions/344. Reverse String/344.py
class Solution:
    def reverseString(self, s: List[str]) -> None:
        start = 0 
        end = len(s) - 1
        while start <= end:
            temp = s[end]
            s[end] = s[start]
            s[start] = temp
            start += 1
            end -= 1
        return s
        